Identity/Identity.DataAccess/Models/Entities/UserEntity.cs

```diff
@@ -0,0 +1,29 @@
+using System.ComponentModel.DataAnnotations.Schema;
+using System.ComponentModel.DataAnnotations;
+
+namespace Identity.API.Models.Entities;
+
+public class UserEntity
+{
+	[Key]
+	[DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+	public int Id { get; set; }
+
+	[Required]
+	public string Username { get; set; }
```

Comment:
There are quite a few warnings here as well. It seems like in Recipes DB pull request Rytis used required keyword rather than attribute.

Reply:
The required attribute makes it `NOT NULL` in SQL. You should also probably specify `public required type` as well

Reply:
Added required keyword